<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextQueryFrame</class>
 <widget class="QDialog" name="TextQueryFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>383</width>
    <height>351</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="addressLabel">
       <property name="text">
        <string>&amp;Address:</string>
       </property>
       <property name="buddy">
        <cstring>addressLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="wordLineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="wordLabel">
       <property name="text">
        <string>&amp;Word:</string>
       </property>
       <property name="buddy">
        <cstring>wordLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="addressLineEdit"/>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>17</width>
       <height>90</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QPushButton" name="queryButton">
       <property name="text">
        <string>Query</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="hiddenButton">
       <property name="text">
        <string>Hidden</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="eixtButton">
       <property name="text">
        <string>Exit</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QPlainTextEdit" name="resultArea"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>hiddenButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>resultArea</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>355</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>eixtButton</sender>
   <signal>clicked()</signal>
   <receiver>TextQueryFrame</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>580</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel">
     <x>838</x>
     <y>63</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
